package.json

```diff
@@ -59,7 +59,7 @@
     "lint-staged": "^13.1.2",
     "postcss": "^8.4.21",
     "postcss-scss": "^4.0.6",
-    "prettier": "^2.8.4",
+    "prettier": "^3.0.0",
```

Comment:
I see that Prettier has upgraded to a major version, and I need to manually execute yarn prettier --write "./**/*.*ts*" "./**/*.*css" to create a commit that adapts to the new formatting rules.

Reply:
Will be updated by https://github.com/Magickbase/nervos-official-website/pull/253